src/components/custom-login-page.js

```diff
@@ -0,0 +1,105 @@
+import { Typography } from "@mui/material";
```

Comment:
Can we rename this file as `student-custom-login-component` since there is another custom login form component for the examiner too.